src/path-optimization/simple-time-parameterization.cc

```diff
@@ -189,6 +180,16 @@ namespace hpp {
         ub = (ub.array() == ub.array()).select(ub,  infinity);
         lb = (lb.array() == lb.array()).select(lb, -infinity);
 
+        for (size_type i=0; i < cb.size(); ++i) {
```

Comment:
I am not sure this will work. If velocity bounds are [-infinity, +infinity],` lb[i]` and `ub[i]` will not be NaN, will they ?

Reply:
But it is not a problem if some coefficients in lb and ub are infinite. It is only a problem if they are all infinite.

Reply:
I understand. Maybe @DianeBury can test that with cogiro ?